src/hotspot/share/gc/shared/referenceProcessor.cpp

```diff
@@ -197,11 +197,8 @@ void ReferenceProcessor::verify_total_count_zero(DiscoveredList lists[], const c
 #endif
 
 ReferenceProcessorStats ReferenceProcessor::process_discovered_references(
-  BoolObjectClosure*            is_alive,
-  OopClosure*                   keep_alive,
-  VoidClosure*                  complete_gc,
-  AbstractRefProcTaskExecutor*  task_executor,
-  ReferenceProcessorPhaseTimes* phase_times) {
+  AbstractClosureContext& closure_context,
```

Comment:
Please fix indendation of parameters according to Hotspot style.

Reply:
Will fix.